### PR TITLE
store/tikv: coprocess streaming tiny fix

### PR DESCRIPTION
**Pull request by tiancaiamao**
Tiny fix:
1. Handle that the remote peer close streaming connection when the first time `Recv()` is called. 
2. Handle that TiKV doesn't set `coprocessor.Response.Range` field when error happens.

@coocood @winkyao @shenli

### CODE REVIEW

**Comment by tiancaiamao** [comment]
So, I've revert the configuration.
PTAL @disksing @coocood

File: store/tikv/client.go [hunk 1]
```diff
@@ -254,7 +256,10 @@ func (c *rpcClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.R
 	var first *coprocessor.Response
 	first, err = copStream.Recv()
 	if err != nil {
-		return nil, errors.Trace(err)
+		if errors.Cause(err) != io.EOF {
+			return nil, errors.Trace(err)
+		}
+		log.Info("copstream returns nothing for the request.")
```
Comment (disksing) [inline store/tikv/client.go]: Is it expected for tikv to close the stream directly?
If so, I think `Debug` is better, otherwise I think `Warn` is better.

Comment (tiancaiamao) [reply store/tikv/client.go]: I'll change it to debug level

File: store/tikv/coprocessor.go [hunk 2]
```diff
@@ -652,7 +656,7 @@ func (it *copIterator) handleCopStreamResult(bo *Backoffer, stream *tikvrpc.CopS
 
 // handleCopResponse checks coprocessor Response for region split and lock,
 // returns more tasks when that happens, or handles the response if no error.
-func (it *copIterator) handleCopResponse(bo *Backoffer, resp *coprocessor.Response, task *copTask, ch chan copResponse) ([]*copTask, error) {
+func (it *copIterator) handleCopResponse(bo *Backoffer, resp *coprocessor.Response, task *copTask, ch chan copResponse, lastResp *coprocessor.Response) ([]*copTask, error) {
```
Comment (zimulala) [inline store/tikv/coprocessor.go]: Could you add the comment for `lastResp`?

File: store/tikv/coprocessor.go [hunk 3]
```diff
@@ -652,7 +656,9 @@ func (it *copIterator) handleCopStreamResult(bo *Backoffer, stream *tikvrpc.CopS
 
 // handleCopResponse checks coprocessor Response for region split and lock,
 // returns more tasks when that happens, or handles the response if no error.
-func (it *copIterator) handleCopResponse(bo *Backoffer, resp *coprocessor.Response, task *copTask, ch chan copResponse) ([]*copTask, error) {
+// if we're handling streaming coprocessor response, lastResp is the last successful response,
+// if we're handling normal request, lastResp and resp is the same.
+func (it *copIterator) handleCopResponse(bo *Backoffer, resp *coprocessor.Response, task *copTask, ch chan copResponse, lastResp *coprocessor.Response) ([]*copTask, error) {
```
Comment (coocood) [inline store/tikv/coprocessor.go]: How about using `lastRange` instead of `lastResp`?
It's easier to understand the purpose of the argument.

**Comment by tiancaiamao** [comment]
PTAL @disksing

**Pull request merged**